astropy/utils/misc.py

```diff
@@ -298,7 +298,7 @@ def _has_hidden_attribute(filepath):
         """
         Returns True if the given filepath has the hidden attribute on
         MS-Windows.  Based on a post here:
-        http://stackoverflow.com/questions/284115/cross-platform-hidden-file-detection
+        https://stackoverflow.com/questions/284115/cross-platform-hidden-file-detection
```

Comment:
This change is unrelated, strictly speaking, but was convenient to fix while I was doing search and replace.